bit_manipulation/quine_macluskey.cpp

```diff
@@ -0,0 +1,404 @@
+/**
+ * @file quine_macluskey.cpp
+ * @author [jang-jaehyuk](https://github.com/jang-jaehyuk)
+ * @brief Implementation of [minimization of
+ * Boolean functions](https://en.wikipedia.org/wiki/Quine-McCluskey_algorithm).
```

Comment:
Please add a detailed description of how the algorithm works, what does it do, etc..
Let us know if you need any help here or in our [**Discord server**](https://discord.gg/c7MnfGFGa6). 😃